src/ontology/mondo-edit.obo

```diff
@@ -331650,8 +331650,10 @@ xref: Orphanet:284979 {source="MONDO:equivalentTo"}
 xref: SCTID:763839005 {source="MONDO:equivalentTo"}
 xref: UMLS:CN202885 {source="MONDO:equivalentTo"}
 xref: UMLS:CN536247 {source="MONDO:equivalentTo"}
+is_a: MONDO:0007947 {source="https://orcid.org/0000-0002-1780-5230"} ! Marfan syndrome
 is_a: MONDO:0017310 {source="Orphanet:284979"} ! Marfan and Marfan-related disorder
```

Comment:
you can exclude this parent too because it is redundant.

Thanks!

Also, would you mind doing a brief search and see if there should be other children of Marfan? Should we have a generic grouping class and separate adult forms?

Reply:
here is a relevant paper:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5266800/

I guess the other form is called classical Marfan?

Reply:
I wrote some comments to Sabrina on Slack but will add them here too:
not sure if neonatal marfan is actually a child of marfan. I think 'Marfan and Marfan-related disorder' is the grouping class and maybe it should stay as it was.

Reply:
@nicolevasilevsky 
According to the paper above, the other form would be "classical Marfan". 
However, it looks like nobody uses "classical Marfan" (for what I found in my research): they call it "Marfan syndrome" or "neonatal Marfan syndrome" (when found in infants). 
Therefore either we have Marfan syndrome, with one child (which would be 'neonatal Marfan syndrome'), or we keep "neonatal Marfan syndrome' as a sister class of "Marfan syndrome".
Thoughts?

Reply:
From the paper above, it says "[neonatal marfan syndrome) is genotypically and phenotypically distinct from the typical Marfan syndrome and carries a poorer prognosis.", which makes me think it is not a child of Marfan. But they do also say it is caused by a variation in FBN1, which is the same as Marfan.

Do you think it is a proper child of Marfan?

I agree with your assessment that we don't need a classical Marfan term.

Reply:
I think 'neonatal Marfan syndrome' is a child of Marfan syndrome. The affected gene is the same, the phenotypes are similar but more severe and more common in the neonatal form. 
Because it is basically the same disease, but a different "type", I would make it a child of Marfan syndrome. And Ada suggested we make this term a child too. However, we can ask the experts again if we want to be sure.

Reply:
sounds good to me! I think we should just go ahead and make it a child. thanks for looking into this!